### PR TITLE
doc: update Matter known issues for 2.1.1 and 2.1.2 releases

### PR DESCRIPTION
**Pull request by markaj-nordic**
Added an issue about invalid ZAP Tool revision
used by Matter samples.

Signed-off-by: Marcin Kajor <marcin.kajor@nordicsemi.no>

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -885,6 +885,19 @@ KRKNWK-6073: Potential delay during FOTA
...
+KRKNWK-15749: Invalid ZAP Tool revision used
+  The ZAP Tool revision used for the |NCS| v2.1.1 and v2.1.2 releases is not compatible with the :file:`zap` files that define the Matter Data Model.
+  This resuls in the ZAP Tool non being able to parse :file:`zap` files from Matter samples.
+
+  **Workaround:** Check out the proper ZAP Tool revision with the following commands, where *<NCS_root_directory>* is the path to your |NCS| installation:
+
+    .. code-block::
+       cd <NCS_root_directory>/modules/lib/matter/
+       git -C third_party/zap/repo/ checkout -f 2ae226
+       git add third_party/zap/repo/
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:899]:
```suggestion
   .. code-block::
  
      cd <NCS_root_directory>/modules/lib/matter/
      git -C third_party/zap/repo/ checkout -f 2ae226
      git add third_party/zap/repo/
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -885,6 +885,19 @@ KRKNWK-6073: Potential delay during FOTA
 Matter
 ======
 
+.. rst-class:: v2-1-2 v2-1-1
+
+KRKNWK-15749: Invalid ZAP Tool revision used
+  The ZAP Tool revision used for the |NCS| v2.1.1 and v2.1.2 releases is not compatible with the :file:`zap` files that define the Matter Data Model.
```
Comment (Damian-Nordic) [inline doc/nrf/known_issues.rst:891]: Mention that the incompatibility concerns only ZAP files from Matter samples in NCS? It's no that some generic Matter Data Model files are wrong.

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (markaj-nordic) [reply doc/nrf/known_issues.rst]: I added the missing line.

**Review by Damian-Nordic — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -885,6 +885,20 @@ KRKNWK-6073: Potential delay during FOTA
 Matter
 ======
 
+.. rst-class:: v2-1-2 v2-1-1
+
+KRKNWK-15749: Invalid ZAP Tool revision used
+  The ZAP Tool revision used for the |NCS| v2.1.1 and v2.1.2 releases is not compatible with the :file:`zap` files that define the Data Model in |NCS| Matter samples.
+  This resuls in the ZAP Tool non being able to parse :file:`zap` files from Matter samples.
```
Comment (Damian-Nordic) [inline doc/nrf/known_issues.rst:892]:
```suggestion
  This resuls in the ZAP Tool not being able to parse :file:`zap` files from Matter samples.
```

**Review by greg-fer — APPROVED** [review]
Please fix the typo from Damian's comment.

**Review by greg-fer — APPROVED** [review]

**Pull request merged**